src/compile/dom/index.ts

```diff
@@ -100,6 +100,10 @@ export default function dom(
 		builder.addBlock(compiler.javascript);
 	}
 
+	if (compiler.options.dev) {
+		builder.addLine(`const __file = ${JSON.stringify(compiler.file)};`);
```

Comment:
It's of course very unlikely that anyone would use a variable called `__file`, but it seems unwise to knowingly introduce potential conflicts, and I don't really see what this provides over simply including `${JSON.stringify(this.compiler.file)}` over when you generate code calling `addLoc` from Element.ts

Reply:
was just thinking it'd look less cluttered than having the filename repeated throughout the component. i guess we have the tools to pick a name that's guaranteed not to conflict, i was just being lazy